entrypoints/services/AppStateService.ts

```
@@ -0,0 +1,65 @@
+    /**
+     * Create or update the app state
+     */
+    static async saveState(
+        state: Partial<Omit<AppState, "schemaVersion">>
+    ): Promise<AppState> {
+        try {
+            const stateRef = doc(db, this.COLLECTION, this.DOCUMENT_ID);
+            const currentState = await this.getCurrent();
+
+            const newState: AppState = {
+                schemaVersion: this.SCHEMA_VERSION,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)